packages/alea-frontend/pages/course-view/[courseId].tsx

```diff
@@ -393,6 +393,7 @@ const CourseViewPage: NextPage = () => {
             )}
             <CommentNoteToggleView
               uri={'TODO ALEA4-N8.1' }
```

Comment:
instead of using `Edit view` to get the slide uri, find the right uri here and pass it on.